app/views/consignment_not_found_page.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import components.gds.{gds_main_template, pageTitle, sectionHeader}
+@import views.components.BackButton
+@import views.Title
+@import views.Title.NO_SECTION
+
+@this(govukLayout: gds_main_template, pageTitle: pageTitle, sectionHeader: sectionHeader)
+
+@(ucr: String)(implicit request: Request[_], messages: Messages)
+
+@govukLayout(
+    title = Title("ileQueryResponse.ucrNotFound.title", NO_SECTION),
+    backButton = Some(BackButton(messages("site.back"), controllers.routes.ViewSubmissionsController.displayPage))) {
```

Comment:
Is this the right page to go back to?  Not the IleQueryController.displayQueryForm()?

Reply:
For sure. When I started the ticket, this page was not yet available. I will change it.